.github/workflows/tests.yml

```diff
@@ -8,7 +8,7 @@ on:
     # always run on pull requests
 jobs:
```

Comment:
I would add two jobs: One Markdown check and one python check. Reason. They are independent. The Markdown check is no pre-condition for the "Ampersand Check"